src/leveled_iclerk.erl

```diff
@@ -549,44 +584,31 @@ safely_log_filescore(PositionList, FN, Score, SW) ->
     leveled_log:log_timer("IC004", [Score, AvgJump, FN], SW).
 
 -spec size_comparison_score(list(key_size() | corrupted_test_key_size()),
```

Comment:
It's a bit confusing that there is a `leveled_inker:filterfun()` type and a `leveled_cdb:filter_fun()`. Sooner or later this is going to be messed up.